### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ndeloof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+Docker has extended its strategic collaboration with Microsoft and Amazon to simplify code
 to cloud application development for developers and development teams by more
-closely integrating with Azure Container Instances (ACI).
+closely integrating with Azure Container Instances (ACI) and Amazon EC2 Container Service (ECS).
 
-The new ACI experience from local VS Code and Docker Desktop development to
-remote deployment in ACI creates a tighter integration between Docker and
-Microsoft developer technologies provides the following productivity benefits to
+The new Cloud experience from local VS Code and Docker Desktop development to
+remote deployment in ACI/ECS creates a tighter integration between Docker and
```
Comment (chris-crone) [inline README.md:11]: Until we have Compose in the API or run support for ECS, the API won't work with ECS

File: README.md [hunk 2]
```diff
@@ -24,20 +24,23 @@ developers:
...
+  * [Why aren't all the existing Docker commands and flags available?](https://github.com/docker/cloud-integration-beta/issues/4)
+  * [Why can't I map container ports?](https://github.com/docker/cloud-integration-beta/issues/5)
+  * [Is Compose v2 supported?](https://github.com/docker/cloud-integration-beta/issues/24)
+* When using an ECS context:  
+  * ECS integration was previously hosted on [docker/ecs-plugin](https://github.com/docker/ecs-plugin)
 
 ## macOS and Windows installation
 
-The ACI integration is built into Docker Desktop **Edge**.
+The Cloud integration is built into Docker Desktop **Edge**.
```
Comment (chris-crone) [inline README.md:43]:
```suggestion
The Cloud integration is built into Docker Desktop Edge and Stable.
```

File: README.md [hunk 3]
```diff
@@ -56,17 +59,17 @@ with a fresh install of Ubuntu 20.04.
 You can install the new CLI using the install script:
 
 ```console
-curl -L https://raw.githubusercontent.com/docker/aci-integration-beta/main/scripts/install_linux.sh | sh
+curl -L https://raw.githubusercontent.com/docker/cloud-integration-beta/main/scripts/install_linux.sh | sh
```
Comment (chris-crone) [inline README.md:62]: We'll need to update the script as well if you want to rename the binaries lower down

File: README.md [hunk 4]
```diff
@@ -56,17 +59,17 @@ with a fresh install of Ubuntu 20.04.
...
 ```
 
 ### Manual install
 
-You can download the Docker ACI Integration CLI from [latest release](https://github.com/docker/aci-integration-beta/releases/latest).
+You can download the Docker ACI Integration CLI from [latest release](https://github.com/docker/cloud-integration-beta/releases/latest).
```
Comment (gtardif) [inline README.md:67]: the Docker Cloud Integration CLI

**Comment by gtardif** [comment]
changes are outdated

**Pull request closed without merging**